solutions/usaco-1015.mdx

```diff
@@ -0,0 +1,234 @@
+---
+id: usaco-1015
+source: USACO Silver 2020 February
+title: Triangles
+author: Albert Zhu
+---
+
+[Original Problem](http://www.usaco.org/index.php?page=viewproblem2&cpid=1015)
```

Comment:
this isn't needed. the problem link will be automatically generated on the solution page.

Reply:
hi i changed this, is it good now?

Reply:
yes, thanks!